docs/test_plans/README.md

```diff
@@ -0,0 +1,28 @@
+# Test Plans
+
+This is a location to record all test plans that are covering celestia's ecosystem.
+
+A Test plan should be good if the following conditions are met:
+
+- Short and simple
+- Introduction
+- In-Scope
+- Out-of-Scope
+- Risks
```

Comment:
risks + mitigation strategies if we need to patch things or can't fully squash a bug due to timelines/etc